package.json

```diff
@@ -14,6 +14,9 @@
     "textlint-rule-common-misspellings": "^1.0.1",
     "textlint-rule-terminology": "^2.1.4"
   },
+  "resolutions": {
+    "minimist": "^1.2.5"
```

Comment:
So I'm not sure if npm supports it but can we do >= so we don't pin any version, just set constraints?

Reply:
yes it does. Updated.